packages/core-ethereum/crates/core-ethereum-db/src/db.rs

```diff
@@ -561,6 +561,28 @@ impl<T: AsyncKVStorage<Key = Box<[u8]>, Value = Box<[u8]>>> HoprCoreEthereumDbAc
         Ok(())
     }
 
+    async fn get_ticket_price(&self) -> Result<U256> {
+        //utils_log::debug!("DB: get_ticket_price");
+        let key = utils_db::db::Key::new_from_str(TICKET_PRICE_KEY)?;
+
+        self.db
+            .get_or_none::<U256>(key)
+            .await
+            .map(|v| v.unwrap_or(U256::zero()))
```

Comment:
let's find something more meaningful, `U256::zero()` makes validations fail

Reply:
Fixed by not returning a U256.